### ISSUE TITLE
Naz and Chaotic frontend issues

### ISSUE THREAD

**Issue by nnamdiarimah** [Rug-Zombie/rug-zombie-frontend#112]
- [x] https://github.com/Rug-Zombie/rug-zombie-frontend/issues/93
- [x] https://github.com/Rug-Zombie/rug-zombie-frontend/issues/110
- [x] https://github.com/Rug-Zombie/rug-zombie-frontend/issues/111
- [x] https://github.com/Rug-Zombie/rug-zombie-frontend/issues/88

When starting a new task make sure your do the following:
- switch to master branch `git checkout master`
- pull the most recent changes `git pull`
- create your new branch `git checkout -b BRANCH_NAME`

This makes things a lot easier later when merging your work. Once a task is complete you can push and create a pull request where I'll give feedback. Once it's ready and merged we'll mark it as complete.
